src/bunit.web/bunit.web.csproj

```
@@ -22,6 +22,7 @@
 		<PackageReference Include="AngleSharp.Css" Version="0.14.0" />
 		<PackageReference Include="AngleSharp.Diffing" Version="0.14.0" />
 		<PackageReference Include="AngleSharp.Wrappers" Version="1.2.0" />
+		<PackageReference Include="Microsoft.Extensions.Localization.Abstractions" Version="3.1.8" />
 	</ItemGroup>
 
```

```suggestion
	</ItemGroup>

	<ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.1'">
		<PackageReference Include="Microsoft.Extensions.Localization.Abstractions" Version="3.1.1" />
	</ItemGroup>

	<ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
		<PackageReference Include="Microsoft.Extensions.Localization.Abstractions" Version="5.0.0-rc.1.*" />
	</ItemGroup>

```

We have to target both .net5 and netstandard2.1